librz/reg/reg.c

```diff
@@ -7,7 +7,7 @@
 RZ_LIB_VERSION(rz_reg);
 
 static const char *types[RZ_REG_TYPE_LAST + 1] = {
-	"gpr", "drx", "fpu", "mmx", "xmm", "ymm", "flg", "seg", NULL
+	"gpr", "drx", "fpu", "mmx", "xmm", "ymm", "flg", "seg", "sys", "pauth", NULL
```

Comment:
Instead of `pauth` here, I suggest more generic `"sec"`. `pauth` is too ARM and implementation specific.